docs/_includes/logo-page.html

```diff
@@ -12,13 +12,18 @@
                         </a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link" href="{{ site.baseurl }}/blog/index.html">
+                        <a class="nav-link" href="{{ site.baseurl }}/0.22.0-RC1/blog/index.html">
```

Comment:
These links will be broken when browsing the docs from the snapshots because the baseurl will be different: https://github.com/lampepfl/dotty/blob/master/doc-tool/src/dotty/tools/dottydoc/DocDriver.scala#L54

Why point to a snapshot version of the blog from the main page anyway?

Reply:
Good point. Updated the links.

The blog only contains RC releases and someone looking at the latest RC releases should see docs in sync with the release.